JLRoutes/JLRoutes.h

```diff
@@ -38,6 +38,12 @@ static NSString *const kJLRoutesGlobalNamespaceKey = @"JLRoutesGlobalNamespace";
 /// Registers a routePattern with default priority (0) in the receiving scheme namespace.
 + (void)addRoute:(NSString *)routePattern handler:(BOOL (^)(NSDictionary *parameters))handlerBlock;
 - (void)addRoute:(NSString *)routePattern handler:(BOOL (^)(NSDictionary *parameters))handlerBlock; // instance method
++ (void)addRoute:(NSString *)routePattern
+       validator:(BOOL (^)(NSDictionary *parameters))validatorBlock
```

Comment:
any interest in `typedef`ing this block signature? (or is that outside the scope of this PR?)